docs/source/models_howto.rst

```diff
@@ -1,405 +0,0 @@
-How to use models
```

Comment:
shall we keep this one?

Reply:
Sure this can go into the deprecated section?